TASK/TaskVehicleDriveToCoordLongrange.md

```diff
@@ -10,12 +10,29 @@ void TASK_VEHICLE_DRIVE_TO_COORD_LONGRANGE(Ped ped, Vehicle vehicle, float x, fl
 
 
 ## Parameters
-* **ped**: 
-* **vehicle**: 
-* **x**: 
-* **y**: 
-* **z**: 
-* **speed**: 
-* **driveMode**: 
-* **stopRange**: 
+* **ped**: Ped id for the task.
+* **vehicle**: Vehicle entity id for the task. 
+* **x**: Destination X coordinate.
+* **y**: Destination Y coordinate.
+* **z**: Destination Z coordinate.
+* **speed**: Speed of driving.
+* **driveMode**: Use 447 to simulate a real-life driver. More info can be found at [Here](https://vespura.com/fivem/drivingstyle/)
+* **stopRange**: Stops in the specific range near the destination. 20.0 works fine.
 
+You can let your character drive to the destination at the speed and driving style you set. You can use map marks to set the destination.
+
+## Examples
+```cs
+// A short example showcasing how this native works with map marks.
+
+// Get the map mark location.
+Vector3 destination = GetBlipInfoIdCoord(GetFirstBlipInfoId(8));
```

Comment:
Note that this may not provide a correct Z coordinate if the blip (for example the Waypoint blip) doesn't have a Z value associated with it. Causing the driving task to have an unexpected destination in case of overlapping roads for example.